.github/workflows/main.yml

```
@@ -31,9 +31,10 @@ jobs:
       - uses: actions/checkout@v2
```

actionlint (1.7.9)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)